src/jemalloc.c

```diff
@@ -1747,18 +1747,23 @@ imalloc_body(static_opts_t *sopts, dynamic_opts_t *dopts) {
 		 * Note that if we're going down this path, usize must have been
 		 * initialized in the previous if statement.
 		 */
+		bool is_large = false;
```

Comment:
How about we move the ```alloc_ctx``` declaration up here, and initialize ```alloc_ctx.slab``` on all paths through the code?  That avoids repeated initialization, and the compiler will typically tell us if we miss a path.